test/functional/testlib.bash

```diff
@@ -0,0 +1,502 @@
+#!bin/bash
+
+FUNC_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+TEST_FILENAME=$(basename "$BATS_TEST_FILENAME")
+TEST_NAME=${TEST_FILENAME%.bats}
+export TEST_NAME
+export FUNC_DIR
+export SWUPD_DIR="$FUNC_DIR/../.."
+export SWUPD="$SWUPD_DIR/swupd"
+
+
+generate_random_content() { 
+
+	local bottom_range=5
```

Comment:
Maybe have
```
            local bottom_range=${1:5}
            local top_range=${2:100}
```
to allow the amount of data required to be passed?

Reply:
sure, I didn't see why this could be necessary... but it doesn't hurt anybody either and will provide flexibility, I'll add that